combustion-android-ble/src/main/java/inc/combustion/framework/ble/device/RepeatedProbeBleDevice.kt

```diff
@@ -156,36 +180,54 @@ internal class RepeatedProbeBleDevice (
     override suspend fun readModelInformation() = uart.readModelInformation()
 
     suspend fun readProbeFirmwareVersion() {
-        Log.d(LOG_TAG, "MeatNet: readProbeFirmwareVersion")
+        val channel = Channel<Unit>(0)
```

Comment:
I can't comment on lines that weren't modified, but the `override`ed calls immediately above this don't call `routeMonitor.activity()`--should they?

Reply:
Good question -- no.  They are just doing a device information service read, so not using the UART.